src/globus_cli/commands/session/show.py

```diff
@@ -94,8 +100,10 @@ def session_show(*, login_manager):
         "  globus whoami\n"
     )
 
+    print(list_data)
```

Comment:
This looks like a debugging print. Let's remove this and add a test.

Reply:
Ah, yes it is